install/zowe-install.sh

```diff
@@ -11,16 +11,17 @@
 ################################################################################
 
 if [ $# -lt 4 ]; then
-  echo "Usage: $0 -i zowe_install_path -h zowe_dsn_prefix"
+  echo "Usage: $0 -i <zowe_install_path> -h <zowe_dsn_prefix> [-l <log_directory>]"
   exit 1
 fi
 
-while getopts "h:i:l:d" opt; do
+while getopts "f:h:i:l:d" opt; do
   case $opt in
     d) # enable debug mode
       # future use, accept parm to stabilize SMPE packaging
       #debug="-d"
       ;;
+    f) LOG_FILE=$OPTARG;;
```

Comment:
I found not like other configuration scripts, `zowe-install-proc.sh` etc, this script is added a `-f` option. I know why we need `-f` for `zowe-copy-to-JES.sh`, but didn't quite get the purpose of here. But I can see it's not harmful at all.

Reply:
Hey Jack - this is to meet the requirement #801 that Onno had during the smpe packaging, so I've not doc'd it as I consider it internal use. Apologies for the lack of clarity, I've add a comment to explain